suite-native/app/src/App.tsx

```diff
@@ -26,21 +27,21 @@ export const App = () => {
                 'zpub6rszzdAK6RuafeRwyN8z1cgWcXCuKbLmjjfnrW4fWKtcoXQ8787214pNJjnBG5UATyghuNzjn6Lfp5k5xymrLFJnCy46bMYJPyZsbpFGagT',
         })
             .then(accountInfo => {
-                console.log('Account info result: ', JSON.stringify(accountInfo, null, 2));
+                console.warn('Account info result: ', JSON.stringify(accountInfo, null, 2));
```

Comment:
please use only `log` everywhere in this file, warn doesn't make sense it this context

Reply:
Yeah I will. This is just a placeholder functionality though and will have further implementation so it really doesn't matter, right?

Reply:
https://github.com/trezor/trezor-suite/pull/5751/commits/1a3fc9a82c5d856cae3c4f42bd5f332ab8e43328